dcos/nodeutil/util_test.go

```diff
@@ -31,7 +36,11 @@ func TestDetectIP(t *testing.T) {
 }
 
 func TestDetectIPFail(t *testing.T) {
-	d, err := NewNodeInfo(&http.Client{}, dcos.RoleMaster, OptionDetectIP("fixture/detect_ip_bad.sh"))
+	detectScript := "fixture/detect_ip_bad.sh"
```

Comment:
Here we could have a func like this:

```
func getFixture(name string) string {
  switch runtime.GOOS {
    case "windows":
      return "fixture\\" + name + ".ps1"
    default:
      return "fixture/" + name + ".sh"
  }
}
```

And then something like

```
detectScript := getFixture("detect_ip_bad")
```

This will help with the readability of the tests and also dry up the code a bit.